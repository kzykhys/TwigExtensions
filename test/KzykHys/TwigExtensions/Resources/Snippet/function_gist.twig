{% set id = 6316404 %}{% set file = 'default.conf' %}
{{ gist(6421704) }}
{{ gist(6316404) }}
{{ gist(6316404, 'default.conf') }}
{{ gist(id, file) }}