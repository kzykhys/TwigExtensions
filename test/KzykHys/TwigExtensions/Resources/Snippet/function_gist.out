<script src="https://gist.github.com/6421704.js"></script>
<script src="https://gist.github.com/6316404.js"></script>
<script src="https://gist.github.com/6316404.js?file=default.conf"></script>
<script src="https://gist.github.com/6316404.js?file=default.conf"></script>